tests/test_voc_format.py

```diff
@@ -55,32 +63,40 @@ def test_colormap_generator(self):
 
         self.assertTrue(np.array_equal(reference, list(VOC.VocColormap.values())))
 
+    @pytest.mark.priority_medium
+    @pytest.mark.reqids(Requirements.REQ_1)
+    @pytest.mark.component
     def test_can_write_and_parse_labelmap(self):
         src_label_map = VOC.make_voc_label_map()
         src_label_map['qq'] = [None, ['part1', 'part2'], ['act1', 'act2']]
         src_label_map['ww'] = [(10, 20, 30), [], ['act3']]
 
-        with TestDir() as test_dir:
+        with TempTestDir() as test_dir:
             file_path = osp.join(test_dir, 'test.txt')
 
             VOC.write_label_map(file_path, src_label_map)
             dst_label_map = VOC.parse_label_map(file_path)
 
             self.assertEqual(src_label_map, dst_label_map)
 
-class TestExtractorBase(Extractor):
+@pytest.mark.components(DatumaroComponent.Datumaro)
+class TTestExtractorBase(Extractor):
```

Comment:
Please avoid random renames of classes and functions. Can we limit the scope of automatically executed symbols?

Reply:
corrected

Reply:
There is at least 3 solutions I found (https://docs.pytest.org/en/6.2.x/goodpractices.html#conventions-for-python-test-discovery, https://stackoverflow.com/questions/24614851/configure-pytest-discovery-to-ignore-class-name):
- Renaming
- Disabling name discovery - having a `pytest.ini` in the root dir with `python_classes = ; python_functions = ;`
- Adding `__test__ = False` in the class definition
- Having `__init__()` defined in the class (but it produces warnings)

I think, options 2 and 3 would work nice for us.

Reply:
I made a mistake in the second way. 

Disabling name discovery - having a pytest.ini in the root dir with 
``` ini
[pytest]
python_classes = 
python_functions = 
```
+ inheriting all the tests from `unittest.TestCase `

This will work the same way `unittest` works. Personally, I'd choose this way.

Reply:
Corrected.